frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/Inbox/index.js

```diff
@@ -37,25 +33,15 @@ const Inbox = ({ tenants, parentRoute }) => {
     userType: defaultType,
     ulb: tenants?.find((tenant) => tenant?.code === tenantId),
   });
+  const hasApprovalRoles = ["ADVOCATE_APPROVER", "ADVOCATE_CLERK_APPROVER"].reduce((res, curr) => {
+    if (!res) return res;
+    res = roles.some((role) => role.code === curr);
+    return res;
+  }, true);
 
-  // delay for fetching it in inbox api
-  // useEffect(() => {
-  //   const timer = () =>
-  //     setTimeout(() => {
-  //       setIsLoading(false);
-  //     }, 1500);
-
-  //   if (state?.isSentBack) {
-  //     setIsLoading(true);
-  //     timer();
-  //     window.history.replaceState({}, "");
-  //   }
-  //   return () => clearTimeout(timer());
-  // }, [state]);
-
-  // if (isLoading) {
-  //   return <Loader />;
-  // }
+  if (!hasApprovalRoles) {
+    history.push(`/${window?.contextPath}/employee/home/home-pending-task`);
```

Comment:
I have addressed the suggested code did some modification since we need to check every roles to be present instead of Array.prototype.some used Array.prototype.some.every